libraries/matchers.rb

```diff
@@ -70,13 +70,4 @@ def graceful_kill_omnibus_service(pkg)
   def once_kill_omnibus_service(pkg)
     ChefSpec::Matchers::ResourceMatcher.new(:omnibus_service, :once, pkg)
   end
-
-  def render_ingredient_config(pkg)
-    ChefSpec::Matchers::ResourceMatcher.new(:ingredient_config, :render, pkg)
-  end
-
-  def add_ingredient_config(pkg)
-    ChefSpec::Matchers::ResourceMatcher.new(:ingredient_config, :add, pkg)
-  end
```

Comment:
Why are these removed? While we're not using them in the specs, since you used `:step_into`, someone who wants to use these in their own specs will need matchers.